addons/project/res_partner.py

```diff
@@ -36,6 +36,7 @@ def _task_count(self, cr, uid, ids, field_name, arg, context=None):
         'task_count': fields.function(_task_count, string='# Tasks', type='integer'),
     }
 
+    @api.cr_uid_id_context
```

Comment:
Is it very likely that Odoo will drop automatic recognition of methods (which their own methods heavily depend on), but will keep a seldom used decorator, to maintain compatibility with v7?

What about going the other way round, make it @api.multi and drop the inconsistently named arguments altogether?